src/main/java/com/synopsys/integration/create/apigen/data/ClassCategories.java

```diff
@@ -109,14 +110,18 @@ private void populateData() {
         populate(VIEW, GENERATED, "ReportContentsView");
         populate(VIEW, GENERATED, "RoleAssignmentView");
         populate(VIEW, GENERATED, "RoleView");
+        populate(VIEW, GENERATED, "RuntimesView");
+        populate(VIEW, GENERATED, "SbomFieldsScopesView");
         populate(VIEW, GENERATED, "ScanBomEntriesView");
         populate(VIEW, GENERATED, "SettingsAnalysisView");
+        populate(VIEW, GENERATED, "ScanMonitorView");
         populate(VIEW, GENERATED, "ScanView");
         populate(VIEW, GENERATED, "ScanReadinessView");
         populate(VIEW, GENERATED, "SsoConfigurationView");
         populate(VIEW, GENERATED, "SystemOauthClientView");
         populate(VIEW, GENERATED, "TagView");
         populate(VIEW, GENERATED, "TokensView");
+        populate(VIEW, GENERATED, "ToolsView");
```

Comment:
Does this line do anything? I'm not seeing a change or usage around ToolsView in the https://github.com/blackducksoftware/blackduck-common-api/pull/50 MR?

Reply:
It fixes import errors in ApiDiscovery.java when doing a Build.